jujugui/static/gui/src/app/components/entity-details/header/header.js

```diff
@@ -71,10 +80,11 @@ class EntityHeader extends React.Component {
     Callback for getting the bundle YAML.
 
     @method _getBundleYAMLSuccess
+    @param {string} bundleUrl Url for the bundle.
```

Comment:
s/{string}/{Object}/
s/Url/URL/

Reply:
Why would this be an object? I'm just passing the URL?